content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -8,30 +8,108 @@ due to theft, damages, human errors, or other factors. As such, inventory adjust
+  specified, this date will default to the 31st of December of the current year.
+- :guilabel:`User`: is the person assigned to the count in the database. This can either be the
+  person physically counting the inventory, or applying the count in the database.
+- :guilabel:`Product Category`: is the category assigned internally to a specific product. Unless
+  otherwise specified (i.e., as **Consumable** or **Rental**), the default *product category* will
+  be *All*.
+- :guilabel:`Available Quantity`: is the quantity of a specific product that is currently
+  available, based on any outstanding/uncompleted sales orders, purchase orders, or manufacturing
+  orders that might change the available quantity once fulfilled.
+- :guilabel:`Accounting Date`: is the date on which the adjustments will be accounted in the Odoo
+  *Accounting* app.
+- :guilabel:`Company`: is the company whose database these inventory adjustments are being made on.
+  The company is listed in the top right corner of the database, next to the user currently logged
+  in.
+
+.. tip::
+   Some columns are hidden by default. To reveal these columns, click the :guilabel:`additional
+   options` button (three dots icon, at the top right of the header row), and select any desired
+   column to reveal by clicking the :guilabel:`checkbox` next to that option.
 
 Add a new line
-==============
+--------------
+
+To create a new inventory adjustment from the :guilabel:`Inventory Adjustments` page, click
+:guilabel:`Create`. Doing so creates a new, blank inventory adjustment line at the bottom of the
+page.
+
+On this blank inventory adjustment line, click the drop-down menu under the :guilabel:`Product`
+column, and select a product from the drop-down menu. If the selected product is tracked using
+either lots or serial numbers, the desired lot or serial number can also be chosen from the
+drop-down menu under the :guilabel:`Lot/Serial Number` column.
+
+Next, set the value in the :guilabel:`Counted Quantity` column to the quantity counted for that
+product during the inventory adjustment process.
+
+.. note::
+   The :guilabel:`Counted Quantity` for new inventory adjustments is set to `0.00` by default. A
+   move with `0.00` :guilabel:`Quantity Done` will be recorded in the product's inventory
+   adjustment history, so it should be set to reflect the actual quantity counted.
+
+To the right of the :guilabel:`Counted Quality` column, the :guilabel:`Scheduled Date` and
+:guilabel:`User` can also be changed via their respective drop-down menus. Changing the scheduled
+date will change the date that the inventory adjustment should be processed on, and selecting a
+responsible user will assign a user to the specific inventory adjustment (for traceability
+purposes).
+
+Once all changes have been made to the new inventory adjustment line, click away from the line.
+Doing so will save the adjustment and move the line to the top of the page.
```

Comment:
As I am thinking about this further- maybe it would be good to explain that you cna have a line listed for each product in each location. That ight not be intuitive? I made the pedal bin in the above example, but it was set to be in Input, not in the WH- so that is why i thought of that.